torch/_inductor/overrides.py

```diff
@@ -473,6 +568,48 @@ def fuse_binary(gm: torch.fx.GraphModule):
     return gm
 
 
+def fuse_binary_inplace(gm: torch.fx.GraphModule):
+    modules = dict(gm.named_modules())
+    for node in gm.graph.nodes:
+        if check_node_is_add_inplace(node) and (
+            len(node.kwargs) != 2 or node.kwargs["alpha"] == 1.0
```

Comment:
We need a better way of checking whether the `add` op has `out` kwarg or not. The check with `len(node.kwargs)` seems not straightforward. Can we check the existence of `out` in kwargs directly?

Reply:
Yes, I updated the check code,